mautic/mautic
# Show email failed and bounced in lead timeline

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2206 
| BC breaks? | N
| Deprecations? | 

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:
When sending an email, if the email is detected as failed (bounce), it's now shown in the "Recent events" timeline of the contact


#### Steps to test this PR:
1. Create a contact with a "hard bounce" email
2. Send an email to this contact
3. Check the timeline : "Email is sent"
4. Launch the **mautic:fetch:email** command to get the bounces back
5. Contact is now tagged as "bounced", and email is now visible in the timeline with a "bounced"
